.travis.yml

```diff
@@ -4,15 +4,29 @@ sudo: required
 
 dist: trusty
 
+services:
+  - docker
+
+cache:
+  directories:
+  - "$HOME/.gradle"
+
 matrix:
   include:
     - os: linux
       jdk: oraclejdk8
       before_install:
        - chmod +x ./gradlew
        - sudo apt-get install libreoffice
-      script:
-       - ./gradlew build
+      jobs:
+        - stage: test
+          script: ./gradlew check -i
+        - stage: Push docker images
+          if: branch = master && type != pull_request
```

Comment:
This needs to happen on a `release*` branch too.

Reply:
Hmm, tell @thijslemmens as well

Reply:
I know, I don't have a release branch yet for docker-postgres

Reply:
But I wonder if it's needed for jodconverter.
Won't change often, I think.

Reply:
Well, the logic in `build.gradle` only configures clean version tags when the branch starts with release